graylog-storage-elasticsearch6/src/main/java/org/graylog/storage/elasticsearch6/TimeRangeFilterQuery.java

```diff
@@ -16,17 +16,17 @@
  */
 package org.graylog.storage.elasticsearch6;
 
-import org.elasticsearch.index.query.QueryBuilder;
 import org.elasticsearch.index.query.QueryBuilders;
+import org.elasticsearch.index.query.RangeQueryBuilder;
 import org.graylog2.plugin.Message;
 import org.graylog2.plugin.Tools;
 import org.graylog2.plugin.indexer.searches.timeranges.TimeRange;
 
 import javax.annotation.Nullable;
 
-public class IndexHelper {
+public class TimeRangeFilterQuery {
```

Comment:
While `TimeRangeFilterQuery` is a much better name than `IndexHelper` I find that it implies an object that can instantiated instead of a static factory. How about `RangeQueryUtil` or `RangeQueryFactory`?

Reply:
:heavy_check_mark: